src/includes/constants.php

```diff
@@ -58,9 +58,10 @@
             }
         }
 
-        // Determine BASE_PATH by assuming that this file is framework/src/Core/Constants.php
-        //  we can then determine the base path
-        $candidateBasePath = rtrim(dirname(dirname(dirname(__DIR__))), DIRECTORY_SEPARATOR);
+        // Determine BASE_PATH by assuming that this file is vendor/silverstripe/framework/src/includes/constants.php
+        // we can then determine the base path
+        $candidateBasePath = rtrim(dirname(__DIR__, 5), DIRECTORY_SEPARATOR);
```

Comment:
What's this change for?

Reply:
Good catch, that's random, did just a normal rebase. It shouldn't be here.

Reply:
removed